lib/auth.js

```diff
@@ -269,11 +269,29 @@ internals.Auth.prototype._authenticate = function (request, next) {
 
         // Check scope
 
+        var expandScope = function (scope) {
```

Comment:
This was because the linter throws a "don't make functions within a loop" warning, so I moved it out of the loop